crates/editor/src/editor.rs

```diff
@@ -1084,55 +1083,44 @@ impl CompletionsMenu {
             }
         }
 
+        let matcher = AhoCorasick::builder()
```

Comment:
Do we need to pull in AhoCorasick to compare a single string case-insensitively? It sounds like a huge overkill; maybe [unicase](https://docs.rs/unicase/latest/unicase/) would be a better fit there?